cmd/billing.go

```diff
@@ -37,6 +44,21 @@ func newBillingCmd(c *config) *cobra.Command {
 		Short: "manage bills",
 		Long:  "TODO",
 	}
+	excelBillingCmd := &cobra.Command{
```

Comment:
should be a new output option, e.g. `-o xlsx` as for yaml and json.

Reply:
Not really, as it generates ALL available billing data in one excel file.

Reply:
But actually csv creation is done  serverside (cloud-api)